bundle/bundle.go

```diff
@@ -16,25 +16,25 @@ import (
 
 // Bundle is a CNAB metadata document
 type Bundle struct {
-	SchemaVersion      string                 `json:"schemaVersion" mapstructure:"schemaVersion"`
-	Name               string                 `json:"name" mapstructure:"name"`
-	Version            string                 `json:"version" mapstructure:"version"`
-	Description        string                 `json:"description" mapstructure:"description"`
-	Keywords           []string               `json:"keywords,omitempty" mapstructure:"keywords"`
-	Maintainers        []Maintainer           `json:"maintainers,omitempty" mapstructure:"maintainers"`
-	InvocationImages   []InvocationImage      `json:"invocationImages" mapstructure:"invocationImages"`
-	Images             map[string]Image       `json:"images,omitempty" mapstructure:"images"`
-	Actions            map[string]Action      `json:"actions,omitempty" mapstructure:"actions"`
-	Parameters         map[string]Parameter   `json:"parameters,omitempty" mapstructure:"parameters"`
-	Credentials        map[string]Credential  `json:"credentials,omitempty" mapstructure:"credentials"`
-	Outputs            *OutputsDefinition     `json:"outputs,omitempty" mapstructure:"outputs"`
-	Definitions        definition.Definitions `json:"definitions,omitempty" mapstructure:"definitions"`
-	License            string                 `json:"license,omitempty" mapstructure:"license"`
-	RequiredExtensions []string               `json:"requiredExtensions,omitempty" mapstructure:"requiredExtensions"`
+	SchemaVersion      string                 `json:"schemaVersion" yaml:"schemaVersion"`
+	Name               string                 `json:"name" yaml:"name"`
+	Version            string                 `json:"version" yaml:"version"`
+	Description        string                 `json:"description" yaml:"description"`
+	Keywords           []string               `json:"keywords,omitempty" yaml:"keywords,omitempty"`
+	Maintainers        []Maintainer           `json:"maintainers,omitempty" yaml:"maintainers,omitempty"`
+	InvocationImages   []InvocationImage      `json:"invocationImages" yaml:"invocationImages,omitempty"`
```

Comment:
This one should not be `omitempty` I guess.

Reply:
Nice catch! Fixed.